From 394f5ebe4d878d4cb9d16e996fb7e6eac1d373de Mon Sep 17 00:00:00 2001
From: Mousumi Jana <mousumix.jana@intel.com>
Date: Tue, 26 Jul 2016 23:50:50 +0530
Subject: [PATCH 14/31] ASoC: Intel: Skylake: Increase the capabilities of
 front end streams

Add more capabilities to the existing streams so as to support
IVI usecases. IVI needs a total of 6 PB and 6 CP streams
that can run simultaneously on 6 different SSPs.

Change-Id: I2d88cd1cf5bb8da0ca1f247971d504ee097fdc99
Signed-off-by: Mousumi Jana <mousumix.jana@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/10568
Reviewed-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
Tested-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>

Conflicts:
	sound/soc/intel/skylake/skl-pcm.c

Conflicts:
	sound/soc/intel/skylake/skl-pcm.c

Conflicts:
	sound/soc/intel/skylake/skl-pcm.c
---
 sound/soc/intel/skylake/skl-pcm.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/sound/soc/intel/skylake/skl-pcm.c b/sound/soc/intel/skylake/skl-pcm.c
index 90389ef..6c7babe 100644
--- a/sound/soc/intel/skylake/skl-pcm.c
+++ b/sound/soc/intel/skylake/skl-pcm.c
@@ -1294,6 +1294,14 @@ struct skl_dsp_notify_ops cb_ops = {
 {
 	.name = "System Pin 2",
 	.ops = &skl_pcm_dai_ops,
+	.capture = {
+		.stream_name = "System Capture 2",
+		.channels_min = HDA_MONO,
+		.channels_max = HDA_8_CH,
+		.rates = SNDRV_PCM_RATE_8000_192000 | SNDRV_PCM_RATE_KNOT,
+		.formats = SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S20_3LE |
+			SNDRV_PCM_FMTBIT_S24_LE | SNDRV_PCM_FMTBIT_S32_LE,
+	},
 	.playback = {
 		.stream_name = "System Playback 2",
 		.channels_min = HDA_MONO,
-- 
1.9.1

